src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,117 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int DEFAULT_CAPACITY = 10;
+    private T[] listContent = (T[]) new Object[DEFAULT_CAPACITY];
```

Comment:
Let's stick to the task description: Make your ArrayList behave the same way as java.util.ArrayList

ArrayList has constructor without any parameters, where you should initialise your array elements = (T[]) new Object[DEFAULT_CAPACITY];